packages/models/src/rooms.ts

```diff
@@ -258,10 +257,11 @@ export class Rooms {
     };
   }
 
-  asPopulatedMember(row: IRoom.PopulatedMemberRow): IRoom.PopulatedMember {
+  async asPopulatedMember(row: IRoom.PopulatedMemberRow): Promise<IRoom.PopulatedMember> {
     return merge(omit(row, "createdAt", "updatedAt"), {
       createdAt: row.createdAt.toISOString(),
       updatedAt: row.updatedAt.toISOString(),
+      online: false // TODO: substitute this workaround
```

Comment:
Let fill a task on clickup about this issue. Feel free to create the task with a proper description.